orchestra/static/orchestra/project-management/project-management.html

```diff
@@ -8,7 +8,7 @@
         <div class="col-lg-12 col-md-12 col-sm-12">
```

Comment:
should you hide the dropdown entirely instead of disabling it, which might make people think the interface is still loading?

Reply:
I realize you're using this to show the project title, etc. Is there a way to show it but without the dropdown?